### PR TITLE
Add grooming/bathroom/appliances/amenities icons

### PR DESCRIPTION
**Pull request by danielbayley**
Related to #1451, #1454 and #1461.

`shave-face` (shavers only socket) related to #1250.

### CODE REVIEW

**Comment by jguddas** [comment]
Faucet would be great as well 🚱🚰

File: icons/bottle-spray.svg [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M22 2h.01" />
+  <path d="M20 5.5h.01" />
+  <path d="M22 9h.01" />
+  <path d="M10 8a2 2 0 0 1 2-2h3c.6 0 1-.4 1-1V3c0-.6-.4-1-1-1H5C3.3 2 2 3.3 2 5c0 .6.4 1 1 1h1a2 2 0 0 1 2 2v2l-2.3 2.3c-.4.4-.7 1.1-.7 1.7v6a2 2 0 0 0 2 2h6a2 2 0 0 0 2-2v-6c0-.6-.3-1.3-.7-1.7L10 10Z" />
+  <path d="M12 2v4" />
+  <path d="M14 6c0 2 0 3 2 4" />
+  <path d="M8 9v9" />
```
Comment (jguddas) [inline icons/bottle-spray.svg:18]: Removed straw due to violating 2px rule.
icon

```suggestion
```

File: icons/bottle-toothbrush-comb.svg [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect width="4" height="4" x="2" y="3" />
+  <path d="M6 7v2c0 .6.1 1.4.2 2L8 20.8v.2c0 .6-.4 1-1 1H3c-.6 0-1-.4-1-1V7" />
+  <path d="M14 2v7l-2 5v8" />
+  <rect width="4" height="6" x="10" y="3" />
```
Comment (jguddas) [inline icons/bottle-toothbrush-comb.svg:15]:
```suggestion
  <path d="M14 9h-4V3h4" />
```

File: icons/razor.svg [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<svg
```
Comment (jguddas) [inline icons/razor.svg:1]: There is a 2px gap violation for the razor line.

File: icons/shave-face.svg [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<svg
```
Comment (jguddas) [inline icons/shave-face.svg:1]: Hard to identify IMO.

File: icons/toilet-roll.svg [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <ellipse cx="10" cy="8" rx="3" ry="2" />
```
Comment (jguddas) [inline icons/toilet-roll.svg:12]: What about making the center circle smaller?
icon

```suggestion
  <ellipse cx="10" cy="8" rx="2.5" ry="2" />
```

File: icons/towel-folded.svg [hunk 6]
```diff
@@ -0,0 +1,14 @@
+<svg
```
Comment (jguddas) [inline icons/towel-folded.svg:1]: What do you think? icon

**Review by jguddas — APPROVED** [review]
I really like them, they fit super well with the existing icons 👍

File: icons/bottle-dispenser.svg [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <circle cx="18.5" cy="5.5" r=".5" />
```
Comment (karsa-mistmere) [inline icons/bottle-dispenser.svg:12]: This element should be avoided, see https://github.com/lucide-icons/lucide/pull/1436

File: icons/bottle-spray.svg (hunk 1 above)
Comment (karsa-mistmere) [reply icons/bottle-spray.svg:18]: Yeah, it also doesn't scale well: 
image

File: icons/toilet-roll.svg (hunk 5 above)
Comment (karsa-mistmere) [reply icons/toilet-roll.svg:12]: > What about making the center circle smaller?

That would put it off grid and make it blurry at 1x.

**Comment by karsa-mistmere** [comment]
Nice icons, but as per usual, can you please provide some use cases for these?

Some of them are pretty specific to a single item (e.g. `towel-rack-heater`) and I cannot really see any legit use cases for them, apart from being used as a terribly particular product category on an equally specialized shopping website, but Lucide, as an open source icon set, has always been more about providing icons for generic UI needs than being tailored for single-use icons, I think people developing a bathroom appliance web store should bloody well make their own damn icons. 😄

File: icons/bath-bubble.svg [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M15 3h.01" />
+  <circle cx="11.5" cy="6.5" r=".5" />
+  <circle cx="16.5" cy="7.5" r=".5" />
```
[the remark this answers is not in the document]
Comment (jguddas) [reply icons/bath-bubble.svg:14]: Radius can't be .5 or what is the problem?

**Comment by danielbayley** [comment]
> Nice icons, but as per usual, can you please provide some use cases for these?
> 
> Some of them are pretty specific to a single item (e.g. `towel-rack-heater`) and I cannot really see any legit use cases for them, apart from being used as a terribly particular product category on an equally specialized shopping website, but Lucide, as an open source icon set, has always been more about providing icons for generic UI needs than being tailored for single-use icons, I think people developing a bathroom appliance web store should bloody well make their own damn icons. 😄

>

@karsa-mistmere I’m sure there are more… but that particular obscure use case hadn’t occurred to me! I had in mind the much more obvious use of displaying available amenities when e.g booking accommodation (a frequent problem when travelling, especially with a female! 😅 These would save a lot of back-and-forth!), and given that `travel` is a distinct category…


> Related to #1451, #1454 and #1461.

**Comment by danielbayley** [comment]
> can you please provide some use cases for these?

> terribly particular product category on an equally specialized shopping website

> developing a bathroom appliance web store

@karsa-mistmere I’m sure there are more… but that particular obscure use case hadn’t occurred to me; I had in mind the much more obvious use of displaying available amenities when e.g booking accommodation (a frequent problem when travelling, especially with a female! 😅 These would save a lot of back-and-forth!), and given that `travel` is a distinct category…

Maybe `towel-rack-heater` is slightly too specific though…

> Related to #1451, #1454 and #1461.

Same reasoning.

File: icons/shave-face.svg (hunk 4 above)
Comment (danielbayley) [reply icons/shave-face.svg:1]: image

File: icons/towel-folded.svg (hunk 6 above)
Comment (danielbayley) [reply icons/towel-folded.svg:1]: Nice, I slightly prefer the simplicity of the existing one though.

File: icons/toilet-roll.svg (hunk 5 above)
Comment (danielbayley) [reply icons/toilet-roll.svg:12]: > That would put it off grid and make it blurry at 1x.

We need to implement this https://github.com/jguddas/lucide/commit/fa7623c2f433230cecd5ef740784e08b2eff3bc7, also seems like something has gone wrong with the existing preview bot! 😕

File: icons/bath-bubble.svg (hunk 8 above)
[the remark this answers is not in the document]
Comment (danielbayley) [reply icons/bath-bubble.svg:14]: > These elements should be avoided, see #1436

🙈 Is that a rule? It kind of makes this icon (and `palette`). There must be a fix…

**Comment by karsa-mistmere** [comment]
> the much more obvious use of displaying available amenities when e.g booking accommodation…

I don't think I have never in my life seen a heated towel rack listed as an amenity on any booking site, but I think this use case is equally specific. The request still stands, however, can you please provide some use cases for _all_ of these icons?

**Comment by matthewmorek** [comment]
What's blocking this PR from being merged? I'd love to use these in one of my projects.

**Comment by karsa-mistmere** [comment]
> What's blocking this PR from being merged? I'd love to use these in one of my projects.

The same thing as many other pending icon PRs: not all icons meet the standards of our design guidelines, see the discussions above.

**Comment by ericfennis** [comment]
This PR is merged to the lab branch. This PR does not match our [PR template](https://github.com/lucide-icons/lucide/blob/main/.github/pull_request_template.md), which requires valid use cases. To move on we merged this PR to the [lab branch](https://github.com/lucide-icons/lucide/tree/lab), which will be moved to a separate repo in the future.

**Pull request merged**